website/content/docs/configuration/storage/raft.mdx

```diff
@@ -161,6 +161,18 @@ delay) mode. The maximum allowed value is 10.
   Vault's [redundancy zone](/vault/docs/enterprise/redundancy-zones). This is reported to autopilot
   and is used to enhance scaling and resiliency. Use of this feature requires Vault Enterprise.
 
+- `raft_wal` `(boolean: false)` - Setting this to `true` enables the
+  [raft-wal](/vault/docs/internals/integrated-storage#configurable-raft-log-store) log store instead of the default of BoltDB.
```

Comment:
I had multiple mentions of the experimental nature of this change in my first version of this, and Mike's feedback was to pare that down because it was redundant. Rather than going back and forth on this, maybe I'll wait for @schavis to weigh in.